README.md

```diff
@@ -45,14 +45,14 @@ export default {
 
 ### snapshotPath
 
-type: `string`  
+type: `string`
 default: `'.size-snapshot.json'`
 
 ### matchSnapshot
 
-This option allows to check that contributor do not forget to build or to commit `.size-snapshot.json` file. If this is `true` the plugin will match existing snapshot and the new one.
+This option allows you to verify that contributors don't forget to build or commit the `.size-snapshot.json` file. If this is `true`, the plugin will validate the snapshot against an existing one.
 
-type: `boolean`  
+type: `boolean`
```

Comment:
Could you please keep these spaces? They are used in markdown to insert `<br />`.

Reply:
sounds good -- my editor is set to automatically remove trailing whitespaces, so this was on accident :)